MacDown/Code/Preferences/MPTerminalPreferencesViewController.m

```diff
@@ -0,0 +1,226 @@
+//
+//  MPTerminalPreferencesViewController.m
+//  MacDown
+//
+//  Created by Niklas Berglund on 2017-01-11.
+//  Copyright © 2017 Tzu-ping Chung . All rights reserved.
+//
+
+#import "MPTerminalPreferencesViewController.h"
+#import "MPUtilities.h"
+#import "MPPreferences.h"
+
+@interface MPTerminalPreferencesViewController ()
+@property (weak) IBOutlet NSTextField *supportIndicator;
+@property (weak) IBOutlet NSTextField *supportTextField;
+@property (weak) IBOutlet NSTextField *infoTextField;
+@property (weak) IBOutlet NSTextField *locationTextField;
+@property (weak) IBOutlet NSButton *installUninstallButton;
+@end
+
+@implementation MPTerminalPreferencesViewController {
+    NSURL *shellUtilityURL;
+    NSPipe *brewPrefixOutputPipe;
+    NSColor *installedColor;
+    NSColor *notInstalledColor;
+}
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    self->installedColor = [NSColor colorWithDeviceRed:0.357 green:0.659 blue:0.192 alpha:1.000];
+    self->notInstalledColor = [NSColor colorWithDeviceRed:0.897 green:0.231 blue:0.212 alpha:1.000];
```

Comment:
Swift ❤️ 😝
Good point. I totally think it doesn't belong in `viewDidLoad`. What I usually do is set this kind of member variables in the beginning of the `init` method and comment.

The `NS_INLINE` approach is cool too. It's just a bit verbose.